packages/docs/pages/migrationguide.mdx

```diff
@@ -0,0 +1,36 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Migration Guide
+
+Sometimes, a new version of a package will introduce changes which _could be breaking_. We made it part of the development process to document these changes inside the `MIGRATION.md` file of the `app` package. This file will contain a list of changes and how to migrate from the old version to the new version.
+Since developers eventually must manually update the own codebase, we are particularly careful to document these changes and make it as easy as possible to migrate.
```

Comment:
```suggestion
Since developers eventually must manually update their own codebase, we are particularly careful to document these changes and make it as easy as possible to migrate.
```